frontend/src/App.css

```diff
@@ -0,0 +1,75 @@
+.main_div {
+  height: 100vh;
+}
+
+.App {
+  text-align: center;
+}
+
+table,
+th,
+tr,
+td {
+  border: 1px solid black;
+  padding: 4px;
+}
+.buttonTable a{
+  margin: 0 10px;
+}
+.userName{
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.userName h4{
+  margin-bottom: 0;
+}
+.buttonTable a {
+  margin: 0 10px;
+}
+.personBlock {
+  min-width: 420px;
+  max-width: 420px;
+  text-transform: capitalize;
+  border: 1px solid #444444;
+  padding: 16px;
+  border-radius: 16px;
+  margin: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.contriPage {
+  padding: 48px;
+}
+.contriPage>div{
+  padding: 20px 10px;
+}
+.sharePrice {
+  margin-top: 8px;
+  margin-bottom: 0;
+}
+.contriItem {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+.sharePrice {
+  background: #0dcaf0;
+  font-weight: bold;
+  border-radius: 12px;
+  padding: 0 8px;
+}
+.sendEmail{
+  width: 100px;
+  border-radius: 16px;
+  border: 1px solid;
+  margin: 10px 6px;
+}
+.emailInput{
+  border-radius: 4px;
+  padding: 0 4px;
+  margin-right: 10px;
+  border: 1px solid;
+}
```

Comment:
- Its good to see for the common properties you have used class names so that the common CSS can be applied to it. 
- No suggestions, everything looks good to me.